src/aal/aal_loongarch.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+#include <cstddef>
+#if _LOONGARCH_SZPTR == 64
+#  define SNMALLOC_VA_BITS_64
+#elif _LOONGARCH_SZPTR == 32
+#  define SNMALLOC_VA_BITS_32
```

Comment:
These are old-style (MIPS-style) built-in macros; better use `__SIZEOF_POINTER__` which is actually architecture-independent. (See https://github.com/loongson/LoongArch-Documentation/pull/28 for further details.)